test/x-address-api-test.ts

```diff
@@ -3371,7 +3366,7 @@ describe('X-address Usage', function () {
             taker: address
           })
         ]
-      ).then((directOfferResults, reverseOfferResults) => {
+      ).then(([directOfferResults, reverseOfferResults]) => {
```

Comment:
How did this API  move from two parameters to an array of one parameter? Was there an underlying change somewhere that went in without this file being updated? Did this ever work?

Reply:
Yea, this was a bug caught by TS. Promises never return more than one value, so `(directOfferResults, reverseOfferResults)` was a typo where the author meant to pass a single array of the two values.

Looks like this never worked, and isn't currently being run in the test suite. 

I'll continue with the work to fix the tests, now that we have them running and failing in this branch.

Reply:
SGTM - thanks for the explanation.